src/test/a11y/tests/public-routes.test.ts

```diff
@@ -44,6 +44,21 @@ describe('Accessibility - Public Routes', () => {
         describe('A-Z Search Page', () => {
             describe('with no input data', () => {
                 const url = '/search';
+                testAccessibility(url, '', true, { 'search-input': '' });
+            });
+        });
+
+        describe('Sign-in Page', () => {
+            describe('with no input data', () => {
+                const url = '/sign-in';
+                testAccessibility(url, '', true, { 'input-autocomplete': '' });
+            });
+        });
+
+        describe('View option  Page', () => {
+            const url = '/view-option';
+
+            describe('with no input data', () => {
                 testAccessibility(url, '', true, { 'input-autocomplete': '' });
```

Comment:
'input-autocomplete' is not the right post body for this test. Please check the controller code